x/nexus/keeper/migrate.go

```diff
@@ -3,23 +3,23 @@ package keeper
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
+	axelarnettypes "github.com/axelarnetwork/axelar-core/x/axelarnet/types"
 	"github.com/axelarnetwork/axelar-core/x/nexus/types"
 )
 
-// Migrate6to7 returns the handler that performs in-place store migrations
-func Migrate6to7(k Keeper) func(ctx sdk.Context) error {
+// Migrate7to8 returns the handler that performs in-place store migrations
+func Migrate7to8(_ Keeper, bank types.BankKeeper, account types.AccountKeeper) func(ctx sdk.Context) error {
 	return func(ctx sdk.Context) error {
-		addModuleParamGateway(ctx, k)
-		addModuleParamEndBlockerLimit(ctx, k)
+		if err := sendCoinsFromAxelarnetToNexus(ctx, bank, account); err != nil {
+			return err
+		}
 
 		return nil
 	}
 }
 
-func addModuleParamGateway(ctx sdk.Context, k Keeper) {
-	k.params.Set(ctx, types.KeyGateway, types.DefaultParams().Gateway)
-}
+func sendCoinsFromAxelarnetToNexus(ctx sdk.Context, bank types.BankKeeper, account types.AccountKeeper) error {
```

Comment:
what is this migration for? axelarnet module account doesn't store any coins. Are we going to move the per denom escrow account to use the nexus module account instead?

Reply:
is this blocked address [filter](https://github.com/axelarnetwork/axelar-core/blob/decb2a6ed725cd06dbae8631efb0219bbf63e9bd/app/keepers.go#L451) still needed now that the special IBC/GMP account is the sender

Reply:
>  Was this migration supposed to move the per denom escrow balances to the nexus module account instead?
I think the per denom escrow address is not changed.

> is this blocked address [filter](https://github.com/axelarnetwork/axelar-core/blob/decb2a6ed725cd06dbae8631efb0219bbf63e9bd/app/keepers.go#L451) still needed now that the special IBC/GMP account is the sender?

amplifier sends gmp with token to nexus module, we need unblock nexus?